### PR TITLE
DOC/MNT: Add docstring. Update intensity input box

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

**Comment by ericdill** [comment]
Ah wait, don't merge this yet. The limit combo box is not updating

**Comment by ericdill** [comment]
@CJ-Wright Can you try this out and see if it works for you?  I added 4 kwargs to the init of `CrossSectionMainWindow` for `img_min`, `img_max`, `norm` and `intensity_scaling`, which should let you do exactly what you are looking to do.  I went ahead and made these changes because this is a pretty convoluted code path...  I apologize if you were excited about making these changes, but I assumed that you were not

File: xray_vision/messenger/mpl/cross_section_2d.py [hunk 1]
```diff
@@ -48,7 +48,7 @@
 from ...backend.mpl.cross_section_2d import CrossSection2DView
 from ...backend.mpl import cross_section_2d as View
 from ...backend.mpl import AbstractMPLDataView
-
+import pdb
```
Comment (CJ-Wright) [inline xray_vision/messenger/mpl/cross_section_2d.py:51]: Do you still need the debuger?

**Comment by CJ-Wright** [comment]
@ericdill It's fine, I am more interested in the code being finished then getting the chance to do it myself. Thank you for the code, sorry if you felt dragged into this.

**Comment by CJ-Wright** [comment]
@ericdill I am getting an error 

``` python
Traceback (most recent call last):
  File "/mnt/work-data/dev/xpd_workflow/xpd_workflow/vis.py", line 65, in <module>
    tt = StackExplorer(key_list, data_list)
  File "/mnt/work-data/dev/xpd_workflow/xpd_workflow/vis.py", line 27, in __init__
    img_max=.95)
  File "/mnt/work-data/dev/xray-vision/xray_vision/qt_widgets/__init__.py", line 94, in __init__
    self._ctrl_widget.set_min_intensity_limit(img_min)
  File "/mnt/work-data/dev/xray-vision/xray_vision/messenger/mpl/cross_section_2d.py", line 494, in set_min_intensity_limit
    self._spim_min.setValue(min_intensity)
AttributeError: 'CrossSection2DControlWidget' object has no attribute '_spim_min'
```

Thoughts?

**Comment by CJ-Wright** [comment]
Never mind I got it

**Comment by ericdill** [comment]
>  I am more interested in the code being finished then getting the chance to do it myself

Great, that's what I figured.

> Thank you for the code, 

You're welcome!

> sorry if you felt dragged into this.

No worries

> Do you still need the debuger?

Nope, can you delete that import?

**Comment by CJ-Wright** [comment]
I think I did delete that import in the final version

**Pull request merged**